### PR TITLE
Add tommi.space

### PR DESCRIPTION
**Pull request by xplosionmind**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://tommi.space

### CODE REVIEW

File: domains/tommi.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "email": "surfing@tommi.space",
+    "GitHub": "xplosionmind",
+    "GitLab": "xplosionmind",
+    "Mastodon": "@tommi@pan.rent"
+  },
+	"repo": "https://codeberg.org/tommi/tommi.space",
+	"description": "The virtual representation of Tommi’s mind",
+  "record": {
+    "URL": "https://tommi.space",
+    "CNAME": "tommi.space"
```
Comment (CuteDog5695) [inline domains/tommi.json:13]: You cant use CNAME with any other  record (stick to either URL or CNAME)

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/tommi.json (hunk 1 above)
Comment (xplosionmind) [reply domains/tommi.json]: Sorry for the mistake. I am now using the URL record only.
Thanks!

File: domains/tommi.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "owner": {
+    "username": "tommi",
+    "email": "surfing@tommi.space",
+    "GitHub": "xplosionmind",
```
Comment (CuteDog5695) [inline domains/tommi.json:5]: username needs to be your github username, remove github line and replace "tommi" with your github username (also sorry for late reply)

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/tommi.json (hunk 2 above)
Comment (xplosionmind) [reply domains/tommi.json:5]: Is there a way to use tommi.is-a.dev?

**Comment by xplosionmind** [comment]
Please, reopen if possible… PR was finally ok and ready to be merged…

**Pull request closed without merging**